queries.sql

```diff
@@ -13,3 +13,29 @@ SELECT id, name, escape_attempts, weight_kg FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
+COMMIT;
```

Comment:
- [ Required ] I think it would be nice if you make a `SELECT` query to verify if the `species` column was updated before performing the `COMMIT`. In the project requirements, you are required to verify that the change was effected before performing the `COMMIT`. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-09-29 15-20-29](https://github.com/kazim110/Vet-Clinic/assets/16392046/3b6c4617-c5ee-4cd8-b5f5-6a8344ca23a3)